docs/snippets/modules/model_io/output_parsers/get_started.mdx

```diff
@@ -9,7 +9,7 @@ from langchain.llms import OpenAI
 from langchain.chat_models import ChatOpenAI
 
 from langchain.output_parsers import PydanticOutputParser
-from pydantic import BaseModel, Field, validator
+from pydantic import BaseModel, Field, field_validator
```

Comment:
@olegsinavski langchain is only pseudo compatible with pydantic v2. Generally, it's recommended to use the v1 namespace if using pydantic 2 via pydantic.v1.

Code in documentation should not assume which version of the package users have.

```
from langchain.pydantic_v1 import BaseModel, Field, validator
```

Or this:

```
try: 
   from pydantic.v1 import BaseModel, Field, validator
except ImportError:
   from pydantic import BaseModel, Field, validator
```

Reply:
@eyurtsev I guess then all `from pydantic` should be replaced by `from langchain.pydantic_v1` in all the docs I see? I think I'll fix in the cases I encounter in a separate PRs

Reply:
Also, I don't think I should merge this one, right? What is the strategy moving forward though? Is the plan to move to pydantic 2?